src/components/content/order/migrate/ExportServiceData.tsx

```diff
@@ -0,0 +1,10 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+export const ExportServiceData = (): JSX.Element => {
+    const exportDataContentDescription: string = 'The export function is temporarily unavailable.';
```

Comment:
`The export function is not yet implemented` is a better line.

Reply:
accept